lib/opencast-codebuild-stack.ts

```diff
@@ -127,9 +148,21 @@ export class OpencastCodebuild extends cdk.Stack {
         { prefix: '.test-runner-cache' }
       ),
       environmentVariables,
-      environment: computeEnvironment,
+      environment: {
+        computeType: codebuild.ComputeType.LARGE,
+        // This image is generated by the opencast-ecs-images
+        // project when needed. The version must match.
+        buildImage: codebuild.LinuxBuildImage.fromEcrRepository(
+          ecr.Repository.fromRepositoryName(
+            this,
+            'EcrRepository',
+            'hdce/oc-codebuild-environment'),
+            '2.0.0',
```

Comment:
Could this go in the parameter store config?

Reply:
Yes, I think so!